src/test/java/trackitnus/logic/commands/contact/AddContactCommandTest.java

```diff
@@ -256,6 +256,10 @@ public void updateFilteredLessonList(Predicate<Lesson> predicate) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public void sortLesson() {
+            throw new AssertionError("This method should not be called.");
+        }
```

Comment:
This will be changed in a later PR?

Reply:
this is a stub so it will always be like that